### ISSUE TITLE
Windows net package

### ISSUE THREAD

**Issue by pytimer** [shirou/gopsutil#534]
Hi,

i use gopsutil to collect Windows and Linux machine some performance data, but net is not implemented on the Windows.

I write a tool [win-netstat](https://github.com/pytimer/win-netstat) to implement this feature previously. Now i want to only use gopsutil in my project. So i want to implement net package.

I see some code in the net/net_windows.go, if i implement it, is it use proc like below?
```go
procGetExtendedTCPTable = modiphlpapi.NewProc("GetExtendedTcpTable")
procGetExtendedUDPTable = modiphlpapi.NewProc("GetExtendedUdpTable")
```

I was wondering if any work has been started?

**Comment by shirou** [comment]
Thank you! I think no one is working on it. I am very appreciate if you would send a PR !

**Comment by Lomanic** [comment]
Implemented in #549
